src/core/sys/posix/signal.d

```diff
@@ -2019,32 +2019,5 @@ int pthread_kill(pthread_t, int);
 int pthread_sigmask(int, in sigset_t*, sigset_t*);
 */
 
-version( linux )
-{
-    int pthread_kill(pthread_t, int);
-    int pthread_sigmask(int, in sigset_t*, sigset_t*);
-}
-else version( OSX )
-{
-    int pthread_kill(pthread_t, int);
-    int pthread_sigmask(int, in sigset_t*, sigset_t*);
-}
-else version( FreeBSD )
-{
-    int pthread_kill(pthread_t, int);
-    int pthread_sigmask(int, in sigset_t*, sigset_t*);
-}
-else version (Solaris)
-{
-    int pthread_kill(pthread_t, int);
-    int pthread_sigmask(int, in sigset_t*, sigset_t*);
-}
-else version( Android )
-{
-    int pthread_kill(pthread_t, int);
-    int pthread_sigmask(int, in sigset_t*, sigset_t*);
-}
-else
-{
-    static assert(false, "Unsupported platform");
-}
+int pthread_kill(pthread_t, int);
+int pthread_sigmask(int, in sigset_t*, sigset_t*);
```

Comment:
[Both](http://pubs.opengroup.org/onlinepubs/9699919799/functions/pthread_kill.html) [are](http://pubs.opengroup.org/onlinepubs/9699919799/functions/pthread_sigmask.html) defined this way in POSIX standard.